src/IO/ReadSettings.h

```diff
@@ -77,10 +77,13 @@ struct ReadSettings
 
     size_t remote_fs_read_max_backoff_ms = 10000;
     size_t remote_fs_read_backoff_max_tries = 4;
+
     bool enable_filesystem_cache = true;
     size_t filesystem_cache_max_wait_sec = 1;
     bool read_from_filesystem_cache_if_exists_otherwise_bypass_cache = false;
     bool enable_filesystem_cache_log = false;
+    bool is_file_cache_persistent = false;
+    bool enable_filesystem_cache_on_lower_level = true;
```

Comment:
unclear name. What is lower level? nested?

Reply:
added a comment.